data/ideas.json

```
@@ -125,6 +125,19 @@
         "level": "basic",
         "tags": "telegram-bot, bot"
     },
+    {
+        "title": "Read With Me",
+        "description": "A web app, basically an open library that has alot of books and texts available to users. The books are not necessarily owned by the library, a registered user can share their own book with the library where the book/text is unavailable in the library. To attract users, a feature where users can get rewarded with fungible tokens or any worthy collection where a book is read within a particular time or an esteemed user whose texts/books gets a stars for being the most read. Books in this context can be any kind of book, text or collection",
+        "level": "intermediate",
+        "tags": "open-library, web-app"
+
+    },
+    {
+        "title": "A Language Translator",
+        "description": "Just like shazam, an AI application that translates languages or words on the go to the user desired language",
+        "level": "intermediate",
+        "tags": "machine-learning, language-translator, artificial intelligence"
```

The first tag is using as primary key of the idea, so please move `language-translator` to first tag.
```suggestion
        "tags": "language-translator, machine-learning, artificial intelligence"
```